atomic_reactor/plugins/post_push_operator_manifest.py

```diff
@@ -1,172 +0,0 @@
-"""
-Copyright (c) 2019 Red Hat, Inc
-All rights reserved.
-
-This software may be modified and distributed under the terms
-of the BSD license. See the LICENSE file for details.
-"""
-import os
-
-from atomic_reactor.constants import (
-    DEFAULT_DOWNLOAD_BLOCK_SIZE,
-    PLUGIN_PUSH_OPERATOR_MANIFESTS_KEY,
-    OPERATOR_MANIFESTS_ARCHIVE,
-)
-from atomic_reactor.utils.omps import OMPS, OMPSError
-from atomic_reactor.plugin import PostBuildPlugin
-from atomic_reactor.plugins.build_orchestrate_build import get_koji_upload_dir
-from atomic_reactor.plugins.pre_check_and_set_rebuild import is_rebuild
-from atomic_reactor.plugins.pre_reactor_config import (
-    get_omps_config,
-    get_koji_path_info,
-)
-from atomic_reactor.util import (
-    is_isolated_build,
-    is_scratch_build,
-    has_operator_appregistry_manifest,
-    has_operator_bundle_manifest,
-    get_retrying_requests_session,
-)
-
-
-class PushOperatorManifestsPlugin(PostBuildPlugin):
```

Comment:
Could the OMPS removal have its own commit (or maybe its own PR)? And do we want to remove `utils.omps` as well?

Reply:
And docs will need to be updated too.

Reply:
yeah I'll remove utils.omps as well

I'll move it to separate commit

Reply:
moved